packages/grid/x-data-grid/src/components/columnHeaders/GridColumnHeaderItem.tsx

```diff
@@ -156,7 +156,7 @@ function GridColumnHeaderItem(props: GridColumnHeaderItemProps) {
 
   const width = column.computedWidth;
 
-  let ariaSort: 'ascending' | 'descending' | undefined;
+  let ariaSort: 'ascending' | 'descending' | 'none' = 'none';
```

Comment:
Wouldn't just have it as `none` work?

Reply:
Not sure what you mean. One `'none'` is for types and another one is used as a value for `ariaSort`